.travis.yml

```diff
@@ -1,6 +1,3 @@
 language: node_js
-node_js: 
-  - "6"
-
-before_script:
-  - npm install -g grunt-cli
+node_js:
+  - "8"
```

Comment:
How about also specifying 10 (and perhaps 11?)

Reply:
Changed to 10. Its only browser functionality, so I'm okay just testing in the latest.